### PR TITLE
Erie pa

### LINKED ISSUE #233: Erie, Pa
Head Of Finances 
PLichtenwalter@erie.pa.us
Economic Community Development
CGroner@erie.pa.us
Community Liason 
MOutlaw@erie.pa.us
Citizen Relations 
MSala@erie.pa.us
Executive asst. to the Mayor 
MBruce@erie.pa.us
Chief of Police 
DSpizarny@erie.pa.us

Erie City Council 
President James Winarski
Jwinarski@erie.pa.us

Lallen@erie.pa.us
Dbrennan@erie.pa.us
Ebrzezinski@erie.pa.us
Mkeys@erie.pa.us
Kschaaf@erie.pa.us
Mwitherspoon@erie.pa.us

Dear Erie Mayor's Office and City Council Members, 
My name is ______ from _______. Nearly a quarter of Erie's overall budget goes into funding for the Erie Beureau of Police, this is unacceptable. I urge you towards an ethical and equal reallocation of the ERIE expense budget, away from ERIE PD and towards social services and educational services.

### PR DESCRIPTION
**Pull request by ctneal91**
Fixes #233

### CODE REVIEW

File: _emails/erie_pa.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Erie, PA
+permalink: "/erie_pa"
```
Comment (teddywilson) [inline _emails/erie_pa.md:3]: I would just use /erie here, no need to include state as well

Comment (ctneal91) [reply _emails/erie_pa.md]: Done.

Comment (ctneal91) [reply _emails/erie_pa.md]: Also changed the file name from `erie_pa.md` to `erie.md`

**Comment by teddywilson** [comment]
This email is a bit too concise for what we're looking for. Could we solicit some more budgeting info from the original issue?

**Comment by ctneal91** [comment]
I think we should @teddywilson. I will go back to the issue and ask for more

**Pull request merged**